packages/claude-hooks/test-openai-tts.mjs

```
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+/**
+ * Test OpenAI TTS integration with proper event format
+ */
+
+import { spawn } from 'child_process'
+import { fileURLToPath } from 'url'
+import { dirname, join } from 'path'
```

ESLint

[error] Expected "path" to come before "url".

(perfectionist/sort-imports)